global.c

```diff
@@ -0,0 +1,18 @@
+#include <stdio.h>
+
+int main(void)
+{
+   int units;
+   printf("How many pounds to a firkin of butter?\n");
+   scanf("%u", &units);
```

Comment:
```suggestion
   scanf("%d", &units);
```

Since an `int` is signed, you need to use `%d` instead of `%u`.